src/send_email.py

```diff
@@ -0,0 +1,32 @@
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import os
+class EmailToSend():
+    def __init__(self, reciver_email, message, title):
+        self.title = title
+        self.reciver_email = reciver_email
+        self.message = message
+    def send(self):
+        sender_email = os.getenv('EMAIL')
+        sender_password = os.getenv('PASSWORD')
+        smtp_host = os.getenv('SMTP_HOST')
+        smtp_port = os.getenv('SMTP_PORT')
+        from_addr = sender_email
+        to = self.reciver_email  
+
+        msg = MIMEMultipart()
+        msg["from_addr"] = from_addr
+        msg["to"] = to 
```

Comment:
mozda bi bilo bolje kad bi ih sve stavio u BCC kako nebi jedni druge videli ko je u mail-u.
Testiraj to pa javi sta bi.
Ako bi ovako isao, onda bi njih 50 videlo da si svima  njima poslao..